aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -112,6 +112,10 @@ If the custom `WebHostService` code requires a service from dependency injection
 
 [!code-csharp[](windows-service/sample/CustomWebHostService.cs?name=Logging&highlight=7)]
 
+## Proxy server and load balancer scenarios
+
+For services that interact with requests from the Internet or a corporate network, additional configuration might be required when hosting behind proxy servers and load balancers. For more information, see [Configure ASP.NET Core to work with proxy servers and load balancers](xref:host-and-deploy/proxy-load-balancer).
```

Comment:
~~For services that interact with requests from the Internet or a corporate network,~~ in all of these.

Reply:
> in all of these

? confused

I don't follow u here. Services that don't do request-response processing wouldn't be dealing with headers at all, so I call out that the reader can ignore this section in those cases. Is that a poor perspective?

Reply:
You have almost the same blurb in multiple docs. The first phrase can be eliminated as it's redundant with the second phrase "...when hosting behind proxy servers...".

Reply:
I don't think it's redundant. The first part of the sentence signals the reader that if their service doesn't do any request-response processing then they can ignore the linked content. Then, just because a service interacts with requests from the Net or a corpnet, that doesn't mean that the service is behind a proxy/LB. Both have to be true:

1. Service must do request-response stuff
2. Service must be behind a proxy/LB that affects the request

It could be rewritten like this ...

> Services that interact with requests from the Internet or a corporate network and are behind a proxy or load balancer might require additional configuration.

Prefer that version?